src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.iOS/AppleCertificatePal.ImportExport.cs

```diff
@@ -118,6 +118,15 @@ internal static X509ContentType GetDerCertContentType(ReadOnlySpan<byte> rawData
                 // We ignore keyStorageFlags which is tracked in https://github.com/dotnet/runtime/issues/52434.
                 // The keys are always imported as ephemeral and never persisted. Exportability is ignored for
                 // the moment and it needs to be investigated how to map it to iOS keychain primitives.
+                if ((keyStorageFlags & X509KeyStorageFlags.Exportable) == X509KeyStorageFlags.Exportable)
+                {
+                    throw new PlatformNotSupportedException(SR.Cryptography_X509_PKCS12_ExportableNotSupported);
+                }
+                if ((keyStorageFlags & X509KeyStorageFlags.PersistKeySet) == X509KeyStorageFlags.PersistKeySet)
```

Comment:
We don't have a lot of horizontal whitespace guidelines, but for me I'd have a blank line between these blocks because they're disassociated and the blank line makes it clearer.